Gifski/EditVideoViewController.swift

```diff
@@ -401,7 +402,7 @@ final class EditVideoViewController: NSViewController {
 		let heightMinMax = resizableDimensions.heightMinMax
 		widthTextField.minMax = Int(widthMinMax.lowerBound)...Int(widthMinMax.upperBound)
 		heightTextField.minMax = Int(heightMinMax.lowerBound)...Int(heightMinMax.upperBound)
-		loopCountTextField.minMax = 0...9999
+		loopCountTextField.minMax = 0...100
```

Comment:
Can you move the range to https://github.com/sindresorhus/Gifski/blob/5c2d437d162948da78565e663df1d2e8cf5253d6/Gifski/Constants.swift#L7 ? And reuse is in both places.

Reply:
Waiting to push...